thechelle13/tech-client
# refactor to address edit glitch in skills

# Refactor to Address Edit Glitch in Skills

This pull request addresses a glitch in the editing functionality related to skills. The changes include:

- Removal of unnecessary webp files in the assets folder.
- Modification in the EditPostForm component to improve skills editing.

These changes resolve the glitch and improve the overall user experience. Please review and merge at your convenience.

---

### Changes Made
- Deleted: `src/assets/agile2.webp`
- Deleted: `src/assets/api.webp`
- Deleted: `src/assets/software-eng.webp`
- Deleted: `src/assets/software-eng2.webp`
- Deleted: `src/assets/webdev.webp`
- Deleted: `src/assets/webdev2.webp`
- Modified: `src/components/forms/EditPostForm.jsx`

---

#### Testing Instructions
Ensure that the glitch in the skills editing is resolved. Test the EditPostForm component to confirm the changes have the intended effect.
